src/views/ContactDetails.js

```diff
@@ -58,6 +75,30 @@ module.exports = kind({
             model: DetailsModel,
             instanceAllRecords: true
         },
+        {
+            name: 'applicationManagerOpenSrvc',
+            kind: LunaService,
+            service: 'luna://com.palm.applicationManager',
+            method: 'open',
+            onResponse: 'serviceResponse',
+            onError: 'serviceError'
+        },
+        {
+            name: 'applicationManagerLaunchSrvc',
+            kind: LunaService,
+            service: 'luna://com.palm.applicationManager',
+            method: 'launch',
+            onResponse: 'serviceResponse',
+            onError: 'serviceError'
+        },
+        {

```

Comment:
Not sure this is the approach we want to follow. I think it might be easier for us to implement to call the phone app with a launchparam that will do the dial. Doing it through com.palm.telephony would require some C++ coding probably. @Tofee what are your thoughts?